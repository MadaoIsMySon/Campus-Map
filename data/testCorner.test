r
0
1
r
0
0
r
1
1
r
Unknown
Unknown
r
Unknown
0
r
1
Unknown
q
