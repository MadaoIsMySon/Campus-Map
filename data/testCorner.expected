First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [0]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [0]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Pittsburgh Building to Pittsburgh Building:
Total distance: 0.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Unknown]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Unknown]
Unknown building: [0]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Unknown]
